app/assets/stylesheets/application.scss

```diff
@@ -1,4 +1,5 @@
 @import "bootstrap-sprockets";
 @import "theme/variables";
 @import "bootstrap";
-@import "theme/bootswatch";
\ No newline at end of file
+@import "theme/bootswatch";
+@import "summary-card";
```

Comment:
Agreed, I wasn't sure about the directory structure we should go with but you just gave it to me. I will update it.